src/main/java/nl/tudelft/dnainator/ui/DNAinator.java

```diff
@@ -24,20 +32,57 @@
 	private static final int MIN_WIDTH = 300;
 	private static final int MIN_HEIGHT = 150;
 
+	private Stage stage;
+	private BooleanProperty ready = new SimpleBooleanProperty(false);
```

Comment:
You should also set the properties' name: `new SimpleBooleanProperty(false "ready");` or something along these lines.